### PR TITLE
Add simple example

### PR DESCRIPTION
**Pull request by dbrgn**
Can be run with `cargo run --example simple`.

Also refs #27.

### CODE REVIEW

**Comment by dbrgn** [comment]
The cool thing about examples is that they're built on `cargo test` :)

File: examples/simple.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+use spaceapi_server::api::Optional::{Value, Absent};
+
+
+fn main() {
+
+    // Create new minimal Status instance
+    let status = api::Status::new(
+        "coredump",
+        "https://www.coredump.ch/logo.png",
+        "https://www.coredump.ch/",
```
Comment (rnestler) [inline examples/simple.rs]: Are you sure that you want to use coredump.ch? Why not use example.com?

File: examples/simple.rs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        api::Location {
+            address: Value("Spinnereistrasse 2, 8640 Rapperswil, Switzerland".into()),
+            lat: 47.22936,
+            lon: 8.82949,
```
Comment (rnestler) [inline examples/simple.rs]: Same here.

File: examples/simple.rs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        api::Contact {
+            irc: Value("irc://freenode.net/#coredump".into()),
+            twitter: Value("@coredump_ch".into()),
+            foursquare: Absent,
+            email: Value("vorstand@lists.coredump.ch".into()),
```
Comment (rnestler) [inline examples/simple.rs]: Maybe email@example.com?

File: examples/simple.rs (hunk 1 above)
Comment (dbrgn) [inline examples/simple.rs]: Yeah, why not? :) The official SpaceAPI docs also use a real example.

File: examples/simple.rs (hunk 3 above)
Comment (dbrgn) [inline examples/simple.rs]: I agree about this one, I'll remove the e-mail.

**Comment by rnestler** [comment]
Squash and merge I would say. :+1:

**Pull request merged**